python/pacemaker/_cts/environment.py

```diff
@@ -224,8 +224,7 @@ def _filter_nodes(self):
 
     def _validate(self):
         if not self["nodes"]:
-            print("No nodes specified!")
-            sys.exit(1)
+            raise ValueError("No nodes specified!")
```

Comment:
I'm thinking UsageError

Reply:
Yeah, I like that one better.

Reply:
There is no UsageError.  argparse does raise ArgumentError for most problems.  Of course, I could also add any exception type we want.

Reply:
Switched back to ValueError.  As this library develops, we might end up adding our own exception classes and we can revisit this at that time.